File: app/src/main/java/com/zydcc/zrdc/location/MyLocationDataSource.java
package com.zydcc.zrdc.location;

import com.esri.arcgisruntime.location.LocationDataSource;
import com.esri.arcgisruntime.location.LocationDataSource.Location;

/**
 * =======================================
 *
 * Create by ningsikai 2020/6/18:11:02 AM
 * ========================================
 */
public class MyLocationDataSource
        extends LocationDataSource
{
    public void UpdateLocation(Location location)
    {
        updateLocation(location);
    }

    protected void onStart()
    {
        onStartCompleted(null);
    }

    protected void onStop() {}
}

File: app/src/main/java/com/zydcc/zrdc/utils/MyLocationDataSource.kt
package com.zydcc.zrdc.utils

import com.esri.arcgisruntime.location.LocationDataSource

/**
 * =======================================
 *
 * Create by ningsikai 2020/7/2:3:01 PM
 * ========================================
 */
class MyLocationDataSource : LocationDataSource() {

    fun UpdateLocation(location: Location) {
        updateLocation(location)
    }


    override fun onStop() {

    }

    override fun onStart() {
        onStartCompleted(null)
    }

}